### DISCUSSION TITLE
순간이동 기능을 구현하고 싶은데 어떻게 하면 좋을까요?

### DISCUSSION THREAD

**Discussion by hipsc159** [naverz/zepeto-studio-kor#243] [5. Q&A]
이제 막 제페토 개발을 시작하고 있습니다.

순간이동 기능을 구현하고 싶은데

import { Collider, GameObject } from 'UnityEngine';
import { ZepetoScriptBehaviour } from 'ZEPETO.Script'

export default class TeleportBlock extends ZepetoScriptBehaviour {

    public teleportPos: GameObject;

    OnTriggerEnter(obj: Collider) {
        this.Teleport(obj.gameObject)
    }

    Teleport(obj: GameObject) {
        obj.transform.position = this.teleportPos.transform.position;
    }
}

하지만 아무 일도 일어나지 않습니다.. 

혹시 이유를 알고 계시는 능력자분 있을까요 ㅜㅜ 

다른 방법으로 구현되도 좋습니다!

**Comment by x4cf** [comment]
혹시 도움이 될까요?
https://github.com/naverz/zepeto-studio-kor/discussions/150

**Comment by hipsc159** [comment]
네 정말 감사합니다!! 제페토에서 지원하는 함수들은 어디에서 확인할 수 있을까요?

**Reply by x4cf** [reply]
https://studio.zepeto.me/guides/creating-your-world
제페토 공식 월드 가이드를 추천드려요 :)
